src/styles.ts

```diff
@@ -51,14 +53,17 @@ export const styles = css`
 		line-height: var(--line-height);
 		font-family: var(--font-family);
 	}
-
+	:host:not(:first-child) {
+		margin-top: var(--margin-top-not-first);
+	}
```

Comment:
This code has no place here. You must position items from the component that uses this component (from the outside). In the story you did it correctly: defined the margin-top at the story level.

Reply:
Oh, ok! Since the first review I thought I had to link everything to some kind of var, but it actually makes sense.

Addressed